src/agent/adu_core_interface/CMakeLists.txt

```diff
@@ -21,12 +21,15 @@ find_package (Parson REQUIRED)
 
 target_link_libraries (
     ${PROJECT_NAME}
-    PUBLIC Parson::parson aduc::c_utils
+    PUBLIC aduc::adu_types
+           aduc::c_utils
+           aduc::communication_abstraction
+           Parson::parson
     PRIVATE aduc::platform_layer
             aduc::pnp_helper
-            aduc::logging
+            aduc::hash_utils

```

Comment:
nit: reorder the libs

Reply:
cmakeformat should do that, i think

---
In reply to: [591968339](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591968339) [](ancestors = 591968339)

Reply:
cmake format does NOT do that. 

Much to my frustration.

---
In reply to: [591974555](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591974555) [](ancestors = 591974555,591968339)